### PR TITLE
Add swagger to schools API

### PR DESCRIPTION
**Pull request by mwharrison**

### CODE REVIEW

**Comment by mwharrison** [comment]
@jpslav - this isn't quite done (a test is still failing) - would you mind taking a look and making sure I am on the right path with Swagger?

**Comment by mwharrison** [comment]
@jpslav just another reminder to take a look at this :)
Most important file being this one that generates the swagger docs: spec/integration/schools_v1_spec.rb

File: coverage/.last_run.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (edwoodward) [inline coverage/.last_run.json:1]: @mwharrison Did you mean to add the coverage files?

**Review by edwoodward — APPROVED** [review]

File: coverage/.last_run.json (hunk 1 above)
Comment (mwharrison) [reply coverage/.last_run.json:1]: I did not, these should be ignored. Let me see if I can remove them.

**Pull request merged**